### PR TITLE
Ir/ednx/bc 19

### PR DESCRIPTION
**Pull request by cocococosti**
Adding eduNext custom requirements.

### CODE REVIEW

File: requirements/edx/development.txt [hunk 1]
```diff
@@ -4,6 +4,9 @@
 #
 #    make upgrade
 #
+
+# checksum change
```
Comment (morenol) [inline requirements/edx/development.txt]: @jfavellar90  is this necessary?

Comment (cocococosti) [reply requirements/edx/development.txt]: this is the commit that adds the checksum comment https://github.com/eduNEXT/edunext-platform/pull/253/commits/08b3f0923043cf616b4d290b293d5b8cfa8d388c

Comment (morenol) [reply requirements/edx/development.txt]: Yes, but I think that we can remove this comment since we do not do any other change in this file.

Comment (cocococosti) [reply requirements/edx/development.txt]: I agree.

**Comment by morenol** [comment]
@cocococosti can you rebase this?

**Comment by jfavellar90** [comment]
@cocococosti please rebase this to make tests passing

**Comment by jfavellar90** [comment]
@cocococosti where these xblock versions are coming from?

**Comment by cocococosti** [comment]
> @cocococosti where these xblock versions are coming from?

which ones exactly? @jfavellar90 
There were a few commits that updated xblock versions, like https://github.com/eduNEXT/edunext-platform/commit/d7a1b2b4184ac510002faff55079f7b61b7cae3e

File: requirements/edx/development.txt (hunk 1 above)
Comment (jfavellar90) [reply requirements/edx/development.txt]: @cocococosti please remove this

File: requirements/edx/base.txt [hunk 2]
```diff
@@ -224,8 +224,8 @@ simplejson==3.16.0        # via django-rest-swagger, dogapi, mailsnake, sailthru
 singledispatch==3.4.0.3
 six==1.11.0
 slumber==0.7.1            # via edx-rest-api-client
-social-auth-app-django==2.1.0
-social-auth-core==1.7.0
+social-auth-app-django==3.1.0
```
Comment (jfavellar90) [inline requirements/edx/base.txt]: I thought it had been already applied on upstream. But even master has the old version. Well, we'll keep ours

File: requirements/edunext/base.txt [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Requirements for edunext.co
```
Comment (jfavellar90) [inline requirements/edunext/base.txt]: What I want you to check here is the fact that these dependencies below are up to date. Normally they don't change so often but just to make sure. Please check this xblocks are not included in the edx default requirements.

File: requirements/edunext/base.txt [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+# Stanford xblocks
+-e git+https://github.com/Stanford-Online/xblock-free-text-response@release/v0.4.0#egg=xblock-free-text-response==0.4.0
+-e git+https://github.com/Stanford-Online/xblock-in-video-quiz.git@release/v0.1.7#egg=invideoquiz_xblock==0.1.7
+
+###################
+# eduNEXT plugins #
+###################
+# plugins must be editable for the settings to appear on the sys path during loading
+-e git+https://github.com/eduNEXT/eox-tenant.git@v0.4.0#egg=eox-tenant==0.4.0
```
Comment (jfavellar90) [inline requirements/edunext/base.txt]: we should use the version 0.6.1 of eox-tenant

File: requirements/edx/base.txt (hunk 2 above)
Comment (morenol) [reply requirements/edx/base.txt]: There is a PR on upstream https://github.com/edx/edx-platform/pull/21183

Comment (jfavellar90) [reply requirements/edx/base.txt]: @morenol cool

**Comment by cocococosti** [comment]
Should I squash this last commit? Could you check it first please? @jfavellar90

File: requirements/edx/development.txt [hunk 5]
```diff
@@ -4,6 +4,7 @@
 #
 #    make upgrade
 #
+
```
Comment (morenol) [inline requirements/edx/development.txt]: please remove this line

Comment (cocococosti) [reply requirements/edx/development.txt]: done

File: requirements/edx/base.in [hunk 6]
```diff
@@ -34,6 +34,7 @@ boto==2.39.0                        # Deprecated version of the AWS SDK; we shou
 boto3==1.4.8                        # Amazon Web Services SDK for Python
 botocore==1.8.17                    # via boto3, s3transfer
 celery==3.1.25                      # Asynchronous task execution library
+ddt==0.8.0                          # via xblock-drag-and-drop-v2 (which probably shouldn't require it)
```
Comment (jfavellar90) [inline requirements/edx/base.in]: I think this ddt dependency is no required anymore

**Comment by jfavellar90** [comment]
@cocococosti Ok it looks good to me. Don't squash it because I want to know the old versions of the xblocks

**Pull request merged**